rollback.sh

```
@@ -0,0 +1,150 @@
+#   DATA_VOLUME_PATH=./data ./rollback.sh --dry-run
+
+set -euo pipefail
+
+DRY_RUN=false
+
+# Configuration
+export DATA_VOLUME_PATH="${DATA_VOLUME_PATH:-/mnt/volume_nyc3_01}"
+DOCKER_COMPOSE_BACKUP="${DATA_VOLUME_PATH}/docker-compose.yaml.backup"
+ENV_BACKUP="${DATA_VOLUME_PATH}/.env.backup"
+DOCKER_COMPOSE="${DATA_VOLUME_PATH}/docker-compose.yaml"
+ENV_FILE="${DATA_VOLUME_PATH}/.env"
```

Shellcheck (0.11.0)

[warning] ENV_FILE appears unused. Verify use (or export if used externally).

(SC2034)